pipeline/sj_luigi_luigi.py

```diff
@@ -110,8 +110,9 @@ def run(self):
         # creates EmailMessage object
         msg = EmailMessage()
         msg["Subject"] = self.subject
+        msg["Date"] = datetime.now() -+ timedelta(hours=8)
```

Comment:
Can you explain to me why both of you using the same -+?

Reply:
We discussed together yesterday so we are using the same code for this part. And regarding to the -+, I think it should be - only, that might be a typo, but the output is fine so both of us didn't realize that.

Reply:
I see. OK.